### DISCUSSION TITLE
Since Google voice interface cannot be used, why not use Youdao or Baidu voice interface?

### DISCUSSION THREAD

**Discussion by wanghaora** [YiiGuxing/TranslationPlugin#2293] [Q&A]
Since Google voice interface cannot be used, why not use Youdao or Baidu voice interface?
http://dict.youdao.com/dictvoice?audio=hello
既然谷歌发声接口不能用，为什么不用有道或者百度的发声接口？

**Comment by YiiGuxing** [comment]
本来是有相应的计划排着的，没想到事发突然，现在我最需要的就是时间啊~  请给我一些时间。。。

**Reply by wanghaora** [reply]
大佬我爱你
